tests/schemas/invalid/test_text_field_typeahead_invalid_suggestions_url.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
We should remove 'typeahead' from the filename. Typeahead is an implementation detail of how we're using the suggestions url in runner.